pkg/build/controller/factory/factory.go

```diff
@@ -98,7 +98,7 @@ func (factory *BuildControllerFactory) Create() controller.RunnableController {
 			err := buildController.HandleBuild(build)
 			if err != nil {
 				// Update the build status message only if it changed.
-				if msg := err.Error(); build.Status.Message != msg {
+				if msg := formatError(err.Error()); build.Status.Message != msg {
```

Comment:
I'm not familiar with what err.Error() returns, what happens when there isn't one, does it return empty string?  or is formatError potentially going to blow up somewhere